lib/get/getCache.js

```diff
@@ -40,7 +58,7 @@ function _copyCache(node, out, fromKey) {
                 var isUserCreatedcacheNext = !node[$modelCreated];
                 var value;
                 if (isObject || isUserCreatedcacheNext) {
```

Comment:
This was another part of the original implementation I didn't understand the motivation for.

I'm not sure what the cacheNext.value === 'object' check was trying to do. We didn't end up cloning cacheNext.value anyway (that is, we didn't deep clone the wrapper in this case).

It seems to imply we only want to maintain the wrapper, for values which are objects (or for nodes which are pre-wrapped by the user), and I don't understand the motivation for the 'values which are objects' decision.